pages/volunteer_role.tsx

```diff
@@ -46,14 +46,18 @@ const VolunteerRolePage = () => {
           paddingLeft: '1rem',
         }}
       >
-        {content.map((item: string, i) => (
-          <li
-            style={{ display: 'list-item', paddingLeft: '1rem', marginBottom: '1rem' }}
-            key={`${i}-${item}`}
-          >
-            {item}
-          </li>
-        ))}
+        {content.map((item: string, i) => {
```

Comment:
This is good.  A more "elegant" way would be to use a filter.
```
content
  .filter(i => it && !whiteSpaceOnlyRegex.test(item))
  .map(i +> <li.......
```

In fact, the filtering could be done in the getSplitFieldRecord function..